### DISCUSSION TITLE
Change of editor interface structure.

### DISCUSSION THREAD

**Discussion by frkntlr** [godotengine/godot-proposals#8762] [Editor]
Greetings,
There are many proposals for changes to the interface, but I see that there are problems with it. That is why I am opening a topic with such a title. 

I don't know much about the construction, but I want to write my own suggestions.

- We should be able to separate any place in the interface system and place it wherever we want. 
- We should be able to work in 2 windows by adding to the parts we have separated.
- We should be able to use the gameplay part both on the editor and in a window.
- We should be able to test the resolution of the gameplay part or by specifying it as a ratio and we should be able to adjust our layout in this way. In the same way, it should be in the editor.
- We should be able to use the Asset browser in the area we want. We should be able to increase the asset size as much as we want and there should be no problem in seeing it.  More features can be brought in, such as star rating and order of importance.

My aim is to discuss this topic further and to make everyone think about what they are lacking. I am opening it for a more general discussion.

**Comment by AThousandShips** [comment]
> We should be able to use the gameplay part both on the editor and in a window.

By this do you mean running the game in the editor? Then there's a proposal for that:
* https://github.com/godotengine/godot-proposals/issues/7213

**Reply by frkntlr** [reply]
I actually opened it for this title and many titles in general.

**Reply by AThousandShips** [reply]
I'm just saying that it's already covered so shouldn't be part of this discussion 🙂
